src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,95 @@
-import { useState } from 'react';
+import { useState, FormEvent, FC } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+interface Props {
+  onAdd: (newMovie: Movie) => void;
+}
+
+export const NewMovie: FC<Props> = ({ onAdd }) => {
+  const [count, setCount] = useState(0);
+  const [newTitle, setNewTitle] = useState('');
+  const [newDescription, setNewDescription] = useState('');
+  const [newImgUrl, setNewImgUrl] = useState('');
+  const [newImdbUrl, setNewImdbUrl] = useState('');
+  const [newImdbId, setNewImdbId] = useState('');
+
+  const isEmpty: boolean = (
```

Comment:
TS already knows about it
```suggestion
  const isEmpty = (
```